collection-scripts/gather_logs

```diff
@@ -1,50 +1,35 @@
-        mkdir -p ${object_collection_path}
-        /usr/bin/oc logs --all-containers --namespace ${ns} ${pod} > "${object_collection_path}/current.log"
-        /usr/bin/oc logs --previous --all-containers --namespace ${ns} ${pod} > "${object_collection_path}/previous.log"
-      done
+# Collect all Pod logs from last 3 days (72 hours) from namespaces where MTC is installed
+for ns in ${namespaces[@]}; do
+  for pod in $(/usr/bin/oc get pods --no-headers --namespace $ns | awk '{print $1}'); do
+    object_collection_path="/must-gather/${cluster}/namespaces/${ns}/logs/${pod}"
+    mkdir -p ${object_collection_path}
+    echo "[cluster=${cluster}][ns=${ns}][pod=${pod}] Collecting Pod logs..."
+    /usr/bin/oc logs --all-containers --since=72h --namespace ${ns} ${pod} > "${object_collection_path}/current.log" &
+    echo "[cluster=${cluster}][ns=${ns}][pod=${pod}] Collecting previous Pod logs..."
+    /usr/bin/oc logs --previous --since=72h --all-containers --namespace ${ns} ${pod} > "${object_collection_path}/previous.log" & 
+  done
 
-      for kind in backup restore; do
-        object_collection_path="/must-gather/${cluster}/namespaces/${ns}/logs/velero-${kind}-logs"
-        mkdir -p ${object_collection_path}
-        export resources=()
-        for i in $(oc -n ${ns} exec $(oc -n ${ns} get po -l component=velero -o custom-columns=name:.metadata.name --no-headers) -- /bin/bash -c "/velero get ${kind}s | grep -v NAME | awk '{ print \$1 }'" | tr -d '\r'); do
-          resources+=($i)
-        done
-        for resource in ${resources[@]}; do
-          oc -n ${ns} exec $(oc -n ${ns} get po -l component=velero -o custom-columns=name:.metadata.name --no-headers) -- /bin/bash -c "/velero ${kind} logs $resource" > "${object_collection_path}/$resource-${kind}.log"
-          oc -n ${ns} exec $(oc -n ${ns} get po -l component=velero -o custom-columns=name:.metadata.name --no-headers) -- /bin/bash -c "/velero ${kind} describe $resource --details" > "${object_collection_path}/$resource-${kind}-describe.log"
-        done
-      done
+  for kind in backup restore; do
+    object_collection_path="/must-gather/${cluster}/namespaces/${ns}/logs/velero-${kind}-logs"
+    mkdir -p ${object_collection_path}
+    export resources=()
+    for i in $(oc -n ${ns} exec $(oc -n ${ns} get po -l component=velero -o custom-columns=name:.metadata.name --no-headers) -- /bin/bash -c "/velero get ${kind}s | grep -v NAME | awk '{ print \$1 }'" | tr -d '\r'); do
+      resources+=($i)
+    done
+    for resource in ${resources[@]}; do
+      echo "[cluster=${cluster}][ns=${ns}] Gathering 'velero ${kind} logs ${resource}'"
+      oc -n ${ns} exec $(oc -n ${ns} get po -l component=velero -o custom-columns=name:.metadata.name --no-headers) -- /bin/bash -c "/velero ${kind} logs $resource --timeout=10s" > "${object_collection_path}/$resource-${kind}.log" &
+      echo "[cluster=${cluster}][ns=${ns}] Gathering 'velero ${kind} describe ${resource}'"
```

Comment:
I added timeouts to `velero backup logs` invocations so that we don't get stuck waiting a minute for logs to come back.